core-primitives/types/src/parentchain.rs

```diff
@@ -40,3 +41,15 @@ pub type BlockHash = sp_core::H256;
 
 /// Alias to 512-bit hash when used in the context of a transaction signature on the chain.
 pub type Signature = MultiSignature;
+
+#[derive(Encode, Decode, Copy, Clone, Debug, PartialEq, Eq)]
+pub enum ParentchainId {
+	/// Primary parentchain containing the teerex pallet.
+	Teerex,
+	/// Some secondary parentchain contaning custom business logic.
+	Secondary,
```

Comment:
secondary could work, but there will be tertiary as well. no need to add this in this PR, but we should have future proof naming

In the spirit of "parent" we could name this "aunt1 chain" or "uncle1 chain". (seriously)

Reply:
so far I didn't come across a use case that would require more than two fully-bidirectional aunt chain connections (a bridge)

However, "secondary doesn't sound intuitive for devs who see their own chain as primary.

For private-tx concept, we called this a "target" chain

what about
* Integritee
* PrimaryTarget
* SecondaryTarget

?

Reply:
concluding:
* Integritee
* TargetA
* TargetB